commons-numbers-fraction/src/test/java/org/apache/commons/numbers/fraction/FractionTest.java

```diff
@@ -463,10 +463,10 @@ void testSubtract() {
     @Test
     void testEqualsAndHashCode() {
         final Fraction zero = Fraction.of(0, 1);
-        Assertions.assertTrue(zero.equals(zero));
-        Assertions.assertFalse(zero.equals(null));
-        Assertions.assertFalse(zero.equals(new Object()));
-        Assertions.assertFalse(zero.equals(Double.valueOf(0)));
+        Assertions.assertEquals(zero, zero);
+        Assertions.assertNotEquals(null, zero);
+        Assertions.assertNotEquals(zero, new Object());
+        Assertions.assertNotEquals(zero, Double.valueOf(0), 0.0);
```

Comment:
This is creating a compile failure in the CI build. Remove the `0.0` tolerance argument. This is a test for Object.equals and not floating-point equivalence of doubles.